docs-src/docs/usingtk.md

```diff
@@ -1,28 +1,38 @@
-## Using tkinter methods
+## Using tkinter
 
-If you are an advanced user, you can still make use of any tkinter method which is not implemented in guizero.
+If you are an advanced user, you can still make use of tkinter when using guizero.
 
-Each guizero widget itself contains a tk widget - you can find out which by looking on the guizero documentation page for the widget. For example, a guizero `TextBox` contains a tkinter `Entry` object. You can always access the internal object using the syntax `<object_name>.tk`.
+You can combine the use of `guizero` and `tkinter` seamlessly in a program, taking advantage of the simplified syntax of guizero whilst still being able to access the full range of functionality in tkinter if you need it.
 
-### Using a tkinter method on a guizero object
+### Using tkinter widgets in guizero
 
-In this example, we have guizero `App` and `TextBox` widgets.
+You can add tk widgets into your guizero app using the `add_tk_widget` method of `App`, `Window` and `Box`.
+
+In this example, we are adding the tkinter widget `Spinbox` into a guizero `App`:
 
 ```python
-from guizero import App, TextBox
+from guizero import App, Text
+from tkinter import Spinbox
+
 app = App()
-name = TextBox(app, text="Laura")
-app.display()
+text = Text(app, text="A Spinbox widget")
+
+spinbox = Spinbox(from_=0, to=10)
```

Comment:
I had to look up `from_` to check it was really a thing. Ugh.

Reply:
Yep...  tkinter has a lot to answer for!